components/inputs/input-text.js

```diff
@@ -0,0 +1,106 @@
+import '../colors/colors.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { inputStyles } from './input-styles.js';
+import { InputTextMixin } from './input-text-mixin.js';
+
+class InputText extends InputTextMixin(LitElement) {
+
+	static get properties() {
+		return {
```

Comment:
I removed `aria-describedby` and `list` for the same reason as `aria-labelledby`